types/luassert/luassert/assert.d.tl

```diff
@@ -198,7 +198,7 @@ local record asserts
    get_level:function(self:asserts, any)
    level:function(self:asserts, integer)
 
-   has_property:function(any, string)
+   has_property:function(any, ?string)
```

Comment:
Looks like this should actually be `function(string, any)`.

The rest looks fine.